template/build/core/phing/tasks/deploy.xml

```diff
@@ -99,11 +106,13 @@
 
     <!-- Copy required files from docroot. -->
     <copy todir="${deploy.dir}/docroot" overwrite="true">
-      <fileset dir="${docroot}">
+      <fileset dir="${docroot}" defaultexcludes="false">
```

Comment:
Which necessary files does setting `defaultexcludes="false"` cause to be included?